.circleci/config.yml

```diff
@@ -22,7 +22,7 @@ jobs:
   # Check that everything (tests, benches, etc) builds using the MSRV
   precheck-msrv:
     docker: &docker
-      - image: jamwaffles/circleci-embedded-graphics:1.56.1-0
+      - image: jamwaffles/circleci-embedded-graphics:1.57.0-0
```

Comment:
It looks like this would require something like https://github.com/embedded-graphics/build-tools/pull/2 for this to work?

Reply:
It would indeed. I can prepare a PR for that once we settle on a version.

Reply:
I've pushed a 1.57.0-0 image to Docker Hub for testing with. This PR should now at least run in CI.